indra/llwebrtc/llwebrtc.cpp

```diff
@@ -1004,6 +1004,7 @@ bool LLWebRTCPeerConnectionImpl::initializeConnection(const LLWebRTCPeerConnecti
             }
 
             webrtc::PeerConnectionInterface::RTCOfferAnswerOptions offerOptions;
+            this->AddRef();
```

Comment:
I don't see the removal anywhere, is this right? This is confusing so might be better to add reason for this as a comment.

Reply:
adding a comment.